docs/web/quickstart.md

```diff
@@ -8,6 +8,7 @@ The following steps are simpler and only need the JRE, a jar file and an OSM fil
  3. Start GraphHopper Maps via: `java -jar *.jar jetty.resourcebase=webapp config=config-example.properties osmreader.osm=berlin-latest.osm.pbf`
  4. After you see 'Started server at HTTP 8989' go to [http://localhost:8989/](http://localhost:8989/) and you should see a map of Berlin. You should be able to click on the map and a route appears.
 
+If you want to setup graphhopper on your own server, we have a separate information page [Deploy](./../core/deploy.md).
```

Comment:
for the graphhopper.sh script you need not only java but also git, bash, wget etc

all easily installed on linux but a lot more complex on others thatswhy there is the "user installation" guide too :)